resources/service.rb

```diff
@@ -2,12 +2,14 @@
 # TODO: migrate to nssm_service with a breaking change notice
 provides :nssm, platform: 'windows'
 
+unified_mode true
+
 property :servicename, String, name_property: true
 property :program, String, required: true
 property :args, String
 property :parameters, Hash, default: lazy { ::Mash.new }
 property :nssm_binary, [String, NilClass], default: lazy { ::NSSM.binary_path node }
-property :start, [TrueClass, FalseClass], default: true
+property :start, [true, false], default: true
```

Comment:
Rubocop fix ? Not sure how it is related to the unified mode commit

Reply:
(same in service noop)

Reply:
Yes, cookstyle